integration_tests/src/main/python/json_matrix_test.py

```diff
@@ -60,12 +60,7 @@ def read_json_as_text(spark, data_path, column_name):
     'spark.rapids.sql.expression.JsonToStructs': 'true',
     'spark.rapids.sql.json.read.float.enabled': 'true',
     'spark.rapids.sql.json.read.double.enabled': 'true',
-    'spark.rapids.sql.json.read.decimal.enabled': 'true',
-    'spark.rapids.sql.json.read.mixedTypesAsString.enabled': 'true'
```

Comment:
Why remove this config: mixedTypesAsString?

Reply:
good catch, it's a merge error, will revert.